### PR TITLE
ACVP: add test case for DRBG

### PR DESCRIPTION
**Pull request by paulidale**
- [x] tests are added or updated

### CODE REVIEW

File: test/acvp_test.c [hunk 1]
```diff
@@ -1292,6 +1292,84 @@ static int rsa_decryption_primitive_test(int id)
...
+    EVP_RAND *rand = NULL;
+    EVP_RAND_CTX *ctx = NULL, *parent = NULL;
+    unsigned char returned_bits[64];
+    const size_t returned_bits_len = sizeof(returned_bits);
+    unsigned int strength = 256;
+    const struct drbg_st *tst  = &drbg_data[id];
+    int res = 0;
+
+    /* Create the seed source */
+    if (!TEST_ptr(rand = EVP_RAND_fetch(libctx, "TEST-RAND", "-fips"))
```
Comment (slontis) [inline test/acvp_test.c:1307]: This test only runs against the fips module.. so "-fips" is optional.

**Review by slontis — COMMENTED** [review]
LGTM..

File: test/acvp_test.c (hunk 1 above)
Comment (paulidale) [reply test/acvp_test.c:1307]: It's required.  The test RNG isn't FIPS and won't be found without this.

**Comment by paulidale** [comment]
The Travis failure isn't relevant that I can tell -- evp_test is failing the sskdf which is unrelated.

**Comment by paulidale** [comment]
ping @openssl/committers

**Comment by t8m** [comment]
Needs rebase

**Comment by paulidale** [comment]
Rebased.

**Comment by paulidale** [comment]
Ping for review ... this isn't essential for beta but should be in before 3.0 release.

**Review by t8m — APPROVED** [review]

**Comment by paulidale** [comment]
Merged to master.
Thanks.

**Pull request closed without merging**